src/assets/styles/themes/light.scss

```diff
@@ -20,13 +20,18 @@
 
   --rp-ui-color-field-bg: var(--rp-ui-base-bg-000);
   --rp-ui-color-field-bg-2: var(--rp-ui-base-bg-100);
+  --rp-ui-color-field-bg-3: var(--rp-ui-base-bg-000);
+  --rp-ui-color-field-bg-3-hover: var(--rp-ui-base-bg-000);
   --rp-ui-color-field-border: var(--rp-ui-base-e-200);
   --rp-ui-color-field-border-2: var(--rp-ui-base-e-300);
   --rp-ui-color-field-border-2-hover: var(--rp-ui-base-e-400);
+  --rp-ui-color-field-border-3: var(--rp-ui-base-e-200);
+  --rp-ui-color-field-border-3-hover: var(--rp-ui-base-e-300);
   --rp-ui-color-field-hover: var(--rp-ui-base-e-300);
   --rp-ui-color-field-hover-2: var(--rp-ui-base-e-400);
   --rp-ui-color-field-opened: var(--rp-ui-base-topaz-pressed);
   --rp-ui-color-field-placeholder: var(--rp-ui-base-almost-black);
+  --rp-ui-color-field-placeholder-3: var(--rp-ui-base-dark-e-300);
```

Comment:
Not needed for light theme?

Reply:
https://www.figma.com/design/gjYQPbeyf4YsH3wZiVKoaj/🛠-RP-DS-6?node-id=149-1360&m=dev